student-work/adam-friedman/api-final-project/README.md

```diff
@@ -0,0 +1,55 @@
+# General Putnam Motel Diner: Orders API
+
+The Orders API allows you to place orders with the kitchen and retrieve a bill based on the orders currently open at a table.
+
+## Base URL
+
+`https://api.gpmdiner.com`
+
+## Endpoints
+The API includes the following endpoints:
+
+Name | Description
+---- | -----------
+[GET /bill](GET-bill.md) | Retrieves a table's bill.
+[POST /order](POST-order-meal.md) | Creates an order and associates it with a table.
+
+## General Response Codes
+
+The API returns the following general response codes. The endpoint descriptions list case-specifc responses.
+
+Code | Description
+---- | -----------
+400  | Bad Request: The request to the server was improperly formatted.
+500  | Internal Server Error: An unexpected server error occured.
+
+
+## API  Workflow
+
+```mermaid
+flowchart TD
+
+A(Begin order) --> B[Select burger meal];
+B --> C[Select patty details:<br> type, weight, cook, quantity];
+C --> D[Select bun type];
+D --> E{toppings?};
+E -- yes -->F[Select toppings];
+F --> G{condiments?};
+E -- no -->G;
+G -- yes --> H[Select condiments];
+H --> I{drink?};
+G -- no --> I;
+I -- yes --> J[Select drink type];
+I -- no --> K[Review order];
+J --> L{ice?};
+L -- yes --> M[Add ice];
+L -- no --> K;
+M --> K;
+K --> N(Submit order);
+```
+
+## References
+I based this project on one designed by [Alex Fiedler](https://www.linkedin.com/feed/update/urn:li:activity:6626465471241732096/).
+
```

Comment:
```suggestion
The diagram used in this API is from [Mermaid](https://mermaid-js.github.io/mermaid/#/)
```

Reply:
While you will think, anyone can open my code and see how I did it, the idea is that the project explains what you used so that the person who is looking at it (from a potential hiring perspective) knows that you know how to do stuff like this in MD.